CMakeLists.txt

```diff
@@ -272,7 +267,7 @@ opm_add_test(flow_blackoil
   LIBRARIES opmsimulators
   SOURCES
   flow/flow.cpp
-  flow/flow_ebos_blackoil.cpp
+  $<TARGET_OBJECTS:flow_libblackoil>
```

Comment:
Was not this target removed?

Reply:
Nope. It's added in the loop (now there's one such target for each flow simulator).

Reply:
Thanks, now I got it!